ffi/koptcontext.lua

```diff
@@ -257,6 +257,45 @@ function KOPTContext_mt.__index:findPageBlocks()
     end
 end
 
+function KOPTContext_mt.__index:getPanelFromPage(pos)
+    if self.src.data then
+        local pixs = k2pdfopt.bitmap2pix(self.src, 0, 0, self.src.width, self.src.height)
+        local pixg
+        if leptonica.pixGetDepth(pixs) == 32 then
+            pixg = leptonica.pixConvertRGBToGrayFast(pixs)
```

Comment:
Does this mean the output will always be grayscale? If so, that may not be desirable.

Reply:
No, grayscale image is converted later to binary image, then we process the binary image to get coordinates of the panels. Then we cut those panels from color image

Reply:
Right now, it cuts from the gray image, but that was just for testing purposes and that part of the code will be removed later, since this method will probably return coordinates of the panels.